### ISSUE TITLE
NVDA reports "not pressed" for some type of controls when activating them

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#5580]
Since 1 or 2 days ago, NVDA started reporting "not pressed", when activating some type of controls. This is most noticeable when pressing space on checkboxes to change their state, but I think it may occur for other type of controls as well.

Steps to reproduce it:
1. Update to the latest "next" snapshot of NVDA.
2. Open one of NVDA's preferences dialogs (e.g. "General Options").
3. Move to a checkbox control and press space to change its state.

Actual results:
NVDA reports the state change, but after that says "not pressed" as well.

Expected result:
NVDA should only report the state change.

Test environment:
- Operating system: Windows 10 Pro (build 10586.17), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- NVDA version: next-12878,004ece4.
- Processor: Intel Core i5-2320 at 3.00GHz.
- RAM Memory: 4.00 GB.
- Graphics: Palit GeForce GT 630, 1024MB dedicated memory + 2047MB shared system memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC662 at Intel Cougar Point PCH - High Definition Audio Controller.

BTW: Isn't this bug related to the improvement made to NVDA so it will report the state change of toggle buttons in newer versions of Windows? And does this improvement apply to toggle buttons in the "Settings" app of Windows 10 as well? Because if it does, it does not work. NVDA reports the state change for toggle buttons in the Notification/Action Center, but not in the "Settings" app.

**Comment by josephsl** [comment]
Hi, yes, this is due to implementation of a ticket. On my system, toggling Tablet Mode does not announce “pressed” state (huge regression). I’ll comment on that ticket, as this one is a duplicate of that ticket. Thanks.

**Comment by Brian1Gaff** [comment]
As it appears several of us raised similar tickets when this problem came to 
be seen on the last  next snap, I'd like to add here that whatever it is 
which is the cause does not involve uia as it also occurs on windows XP.



This account only receives email from the list. Please direct any personal 
email to
briang1@blueyonder.co.uk
Replies to this address vanish into a hole....
Unless Brian Gaff is in the To line display name field.
----- Original Message -----

**Comment by jcsteh** [comment]
Regression caused by f67746dd (#5441).

**Comment by Brian1Gaff** [comment]
Yes this has fixed it in XP as well, many thanks.
Brian
